psychopy/demos/coder/basic/shutdown_key.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+This demo shows how to use a "shutdown key" to
+quit a running experiment. The key does not have
+to be checked manually, but rather gets handled
+automatically by PsychoPy.
+
+This works when using win.flip() (as in this example)
+and wait() (not demonstrated here). If using wait(), be
+sure to set the ``hogCPUperiod`` parameter equal to the
+entire waiting duration.
+"""
+
+from psychopy.visual import Window
+from psychopy.clock import CountdownTimer
+from psychopy.visual.text import TextStim
+from psychopy.preferences import prefs
+
+prefs.general['enableShutdownKey'] = True
+prefs.keys['shutdownKey'] = 'escape'
```

Comment:
Wouldn't it be cleaner to have `prefs.keys['shutdownKeys'] = ['escape', 'f9']` and then an empty list means that no keys will shut it down? Instead of having a separate `enbleShytdownKey` preference that always needs to be followed by setting another variable.

I like the suggestion! It is indeed a pain to check manually.